Customizable recovery payload (#1283)
SumoLogic/sumologic-documentation: docs/alerts/monitors/create-monitor.md

1. Go to the Monitors page by clicking **Manage Data** > **Monitoring** > **Monitors**.
1. Click on the **Add** button > **New Monitor** to add a new Monitor. The creation dialog box will appear.

> #### From a Dashboard

Author: ```suggestion
#### From your Dashboard
```

1. From a Dashboard, hover your mouse over a panel, click the kebab icon, then **Open in Log Search**.
1. From your log search view, click the kebab icon in the upper right corner, then **Create a Monitor**.

> #### From a Log Search

Author: ```suggestion
#### From your Log Search
```

1. Specify the **Trigger Type**. A Monitor can have one critical, warning, and missing data trigger condition, each with one or more notification destinations. Triggers have different options depending on the query and alert type. Click the **Expand** button next to the query type you're using for configuration details.

> <details><summary><strong>Logs Trigger Types</strong> (click to expand)</summary>

Author: ```suggestion
<details><summary><strong>Logs Trigger Types</strong> (expand to view)</summary>
```

</details>

> <details><summary><strong>Metrics Trigger Types</strong> (click to expand)</summary>

Author: ```suggestion
<details><summary><strong>Metrics Trigger Types</strong> (expand to view)</summary>
```

## Step 6: Name and save your monitor

> 1. Enter a **Name** for the Monitor and the **Location** where you want to save it. A **Description** is optional.<br/> ![monitor details during creation.png](/img/monitors/monitor-details-during-creation.png)

Author: ```suggestion
1. Enter a **Name** for the Monitor and the **Location** you want it saved to. A **Description** is optional.<br/> ![monitor details during creation.png](/img/monitors/monitor-details-during-creation.png)
```

### Using Terraform

> You can configure Sumo Logic Monitors using [Terraform modules](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor).

Author: ```suggestion
* You can configure Sumo Logic Monitors using [Terraform modules](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor).
```

> You can also configure Sumo Logic Monitors using [Terraform modules](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor).

Author: ```suggestion
You can configure Sumo Logic Monitors using [Terraform modules](https://github.com/SumoLogic/terraform-sumologic-sumo-logic-monitor).
```
